connect/resources/directory.py

```diff
@@ -47,6 +48,27 @@ def get_asset(self, asset_id):
         text, code = ApiClient(self._config, 'assets/' + asset_id).get()
         return Asset.deserialize(text)
 
+    def list_marketplaces(self, filters=None):
+        """ List the marketplaces.
+
+        :param dict|Query filters: Filters to pass to the request.
+        :return: A list with the marketplaces that match the given filters.
+        :rtype: list[Marketplace]
+        """
+        query = self._get_filters_query(filters, False)
+        text, code = ApiClient(self._config, 'marketplaces' + query.compile()).get()
+        return Marketplace.deserialize(text)
+
+    def get_marketplace(self, marketplace_id):
+        """ Returns the marketplace with the given id.
```

Comment:
Obtains Marketplace object given it's ID

Reply:
Fixed.